lib/roo_on_rails/railties/logging.rb

```diff
@@ -0,0 +1,16 @@
+module RooOnRails
+  module Railties
+    class Logging < Rails::Railtie
+      initializer 'roo_on_rails.logging.before', before: :initialize_logger do
+        require 'roo_on_rails/logger'
+        Rails.logger = config.logger = RooOnRails::Logger.new
+        Rails.logger.debug 'initializer roo_on_rails.logging.before'
+      end
+
+      initializer 'roo_on_rails.logging.after', after: :initialize_logger do
+        Rails.logger.set_log_level
```

Comment:
isn't this called inside `RooOnRails::Logger#initialize` already? Is this here because the `LOG_LEVEL` ENV var could change in the meantime? Well, it's doesn't hurt doing it again I think, this is just a question :-)

Reply:
It is, unfortunately Rails resets the log level based on what's in `config/environments/{development,production}.rb` — and those _will_ contain a loglevel in a fresh Rails apps. Which we want to override :)

Reply:
Yeah the whole logging setup in rails is a bit of a mess.